usig/sgx/usig-enclave.go

```diff
@@ -16,12 +16,66 @@
 
 package sgx
 
-// #cgo pkg-config: $SGX_URTS_PACKAGE
-// #cgo CFLAGS: -I${SRCDIR}/shim
-// #cgo LDFLAGS: -L${SRCDIR}/shim -lusig_shim
-//
-// #include <stdlib.h>
-// #include "usig.h"
+/*
+// We use only a value of `Cflags` from pkg-config.
```

Comment:
"Cflags" looks weird to me , you mean "CFLAGS"?

Reply:
It comes from the notation of [pkg-config](http://manpages.ubuntu.com/manpages/bionic/man1/pkg-config.1.html).

By the directive `#cgo pkg-config: libsgx_urts_sim`, Cgo read values from `/opt/intel/sgxsdk/pkgconfig/libsgx_urts_sim.pc`, which has the following lines:

```
prefix=/opt/intel/sgxsdk
includedir=${prefix}/include
libdir=${prefix}/lib64

Name: libsgx_urts_sim
Description: SGX uRTS simulation library
Version: 2.3.101.46683
Cflags: -I${includedir}
Libs: -L${libdir} -lsgx_urts_sim
Libs.private: -lsgx_uae_service_sim
```

There are options to be given to a compiler and a linker to use this library. In our case, we need only the value of `Cflags`. And the values is the same as in `/opt/intel/sgxsdk/pkgconfig/libsgx_urts.pc`, which is for the hardware mode.

Reply:
OK, thank you for the detail. I have no objection.